ldap-connector-test/src_test/com/axonivy/connector/ldap/test/TestLdap.java

```diff
@@ -56,15 +70,36 @@ static void setupConfig() throws IOException {
       }
     }
     config = JndiConfig.create()
-            .password(password)
-            .url(Ivy.var().get("LdapConnector.Url"))
-            .userName(username)
+            .password(password).url(Ivy.var().get("LdapConnector.Url"))
+            .userName("cn=" + username + "," + DOMAIN_COMPONENT)
             .connectionTimeout(Ivy.var().get("LdapConnector.Connection.Timeout"))
             .provider(Ivy.var().get("LdapConnector.Provider"))
             .referral(Ivy.var().get("LdapConnector.Referral"))
             .toJndiConfig();
     queryExecutor = new LdapQueryExecutor(config);
     writer = new LdapWriter(config);
+    
+    // Setup docker for testing
+    Network network = Network.newNetwork();
+    ldapContainer = new GenericContainer<>(LDAP_IMAGE).withNetwork(network).withNetworkAliases("octopus_ldap")
```

Comment:
any feedback to team Octopus regarding test-containers? @alexsuter

Reply:
using test-containers is a good idea!